tajo-core-tests/src/test/java/org/apache/tajo/ws/rs/resources/TestQueryResultResource.java

```diff
@@ -151,6 +155,25 @@ public void testGetQueryResult() throws Exception {
   }
 
   @Test
+  public void testGetQueryResultWithoutSessionId() throws Exception {
+    String sessionId = generateNewSessionAndGetId();
+    URI queryIdURI = sendNewQueryResquest(sessionId, "select * from lineitem");
+    URI queryResultURI = new URI(queryIdURI + "/result");
+
+    boolean isBadRequest = false;
+    try {
+      GetQueryResultDataResponse response = restClient.target(queryResultURI)
+              .request()
+              .get(new GenericType<>(GetQueryResultDataResponse.class));
+    } catch (BadRequestException e) {
+      isBadRequest = true;
```

Comment:
How about using expected exception of junit? It will make test codes simpler.
Please refer to test cases in TestCatalogAdminClientExceptions.